### ISSUE TITLE
compatibility with Vim versions earlier than 8.0.1630

### ISSUE THREAD

**Issue by Konfekt** [wellle/context.vim#3]
Since 8.0.1630 vim has a built-in `trim(var)` function to strip leading and trailing whitespace from `var`. If support for earlier Vim versions is planned, then `substitute(var,'^\s*\|\s*$','')` will achieve the same.

**Comment by wellle** [comment]
Thank you! This was definitely done by mistake and I'll fix it.

Unfortunately I don't have an old Vim version installed on my machine anymore to test for issues like this. I'll go and try to compile some 7.x version.

I also wish the help pages would mention in which version a function has been added/modified. So I'd be more likely to avoid such recently added functions.

Thank you for this issue, I was about to approve https://github.com/wellle/targets.vim/pull/242 where I suggested to use `trim()` as well, but luckily we can still fix that now before we merge it!
